tests/hla/base_test.py

```diff
@@ -343,22 +343,16 @@ def print_wcs(self, title, wcs):
 
     def read_image(self, filename):
         """
-        Read the image from a fits file
+        Read the image from first extension of a fits file
         """
-        hdu = fits.open(filename)
-
-        image = hdu[1].data
-        hdu.close()
-        return image
+        return fits.getdata(filename,ext=1)
 
     def read_wcs(self, filename):
         """
         Read the wcs of a fits file
         """
-        hdu = fits.open(filename)
-
-        wcs = stwcs.wcsutil.HSTWCS(hdu, 1)
-        hdu.close()
+        from fits.open(filename) as hdu:
+            wcs = stwcs.wcsutil.HSTWCS(hdu, 1)
```

Comment:
I would just replace this with:
```python
wcs = stwcs.wcsutil.HSTWCS(filename, 1)
```

Reply:
and delete `from fits.open(filename) as hdu:` line above

Reply:
^ `from` should be be `with` (and/or these lines can removed as @mcara suggested).